src/box/sql/parse.y

```diff
@@ -290,8 +290,8 @@ cconsname(N) ::= CONSTRAINT nm(X). { N = X; }
 cconsname(N) ::= . { N = Token_nil; }
 ccons ::= DEFAULT term(X).            {sqlAddDefaultValue(pParse,&X);}
 ccons ::= DEFAULT LP expr(X) RP.      {sqlAddDefaultValue(pParse,&X);}
-ccons ::= DEFAULT PLUS term(X).       {sqlAddDefaultValue(pParse,&X);}
-ccons ::= DEFAULT MINUS(A) term(X).      {
+ccons ::= DEFAULT PLUS number(X).       {sqlAddDefaultValue(pParse,&X);}
+ccons ::= DEFAULT MINUS(A) number(X).      {
```

Comment:
Typo: I suggest to save the alignment for the right side.

Reply:
Fixed.